### PR TITLE
Text field password - Show/Hide text

### LINKED ISSUE #78: Text field password - Show/Hide text
# UI

Screenshot 2019-10-29 at 11 24 58
Screenshot 2019-10-29 at 11 25 11
Screenshot 2019-10-29 at 11 25 18

# Usage

In the case when the text field is use for a password, we should have an eye icon on the right allowing the user to click on it to display the password.

```html
<TextField type="password" />

```

# Properties

List all the properties needed for the component:
- **first-click**:  -> show the text
- **second-click**:  -> hide the text

# Slots

none

# Events

none

### PR DESCRIPTION
**Pull request by blackinno**
Closes https://github.com/mesg-foundation/mesg-components/issues/78

### CODE REVIEW

File: packages/TextField/TextField.vue [hunk 1]
```diff
@@ -1,29 +1,47 @@
...
     label: { type: String, required: true },
     placeholder: { type: String, required: true },
     value: { type: String, required: true }
+  },
+  mounted() {
+    this.textType = this.type
+  },
+  data() {
+    return {
+      textType: 'password',
```
Comment (antho1404) [inline packages/TextField/TextField.vue]: Can you try this? If it works you can remove the `mounted` function

```suggestion
      textType: this.type,
```

File: packages/TextField/TextField.vue [hunk 2]
```diff
@@ -1,29 +1,47 @@
 <template>
   <label>
     {{label}}
-    <input
-      :type="type"
-      :placeholder="placeholder"
-      :value="value"
-      @input="$emit(`input`, $event.target.value)"
-    />
+    <div class="password">
```
Comment (antho1404) [inline packages/TextField/TextField.vue]: You should put another class name. When the type is text it doesn't make sense to have a password class

File: packages/TextField/TextField.vue (hunk 1 above)
Comment (blackinno) [reply packages/TextField/TextField.vue]: it works and removes` mounted`

File: packages/TextField/TextField.vue (hunk 2 above)
Comment (blackinno) [reply packages/TextField/TextField.vue]: Fixed.

**Review by antho1404 — APPROVED** [review]

**Pull request merged**